src/tools/DragProbeTool.test.js

```diff
@@ -7,7 +7,7 @@ jest.mock('./../externalModules.js', () => ({
   },
 }));
 
-jest.mock('../import.js', () => ({
+jest.mock('../importInternalModule.js', () => ({
```

Comment:
🤔 
These deps aren't external deps.

Reply:
> No idea actually, I just changed the name but good this was brought up, this is in a lot of places and isn't used, i'll clean it up.